runtime/hydradx/src/xcm.rs

```diff
@@ -480,6 +481,9 @@ pub type LocationToAccountId = (
 	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
 	// Convert ETH to local substrate account
 	EvmAddressConversion<RelayNetwork>,
+	// Converts a location which is a top-level relay chain (which provides its own consensus) into a
+	// 32-byte `AccountId`.
+	GlobalConsensusConvertsFor<UniversalLocation, AccountId>,
```

Comment:
this seems like not Kusama specific, so I assume this will be triggered for Polkadot relay chain too?!

Reply:
no, GlobalConsensus junction is not used when referencing own parent, so this is used currently only with Parents: 2 and a relay chain root at the same time